.travis.yml

```diff
@@ -0,0 +1,14 @@
+language: node_js
+node_js:
+  - "stable"
+install: npm install
+services:
+  - postgresql
```

Comment:
is `sqlite` not better for our test environment, it's lighter, so should make tests run faster

Reply:
I looked into using sqlite and I understand how it will make test run faster. I will look at the travis documentation and implement this feedback.